exercises/D-logical-operators/exercise2.js

```diff
@@ -5,7 +5,21 @@
   Update the code so that you get the expected result.
 */
 
-function isNegative() {}
```

Comment:
Great. Always add a comment for each function as best practice. These functions are simple but for more complex functions, comments assist us when we look back at code so we can determine whether it can be reused for new projects